contributing/community/releases.rst

```diff
@@ -91,7 +91,21 @@ Below is the schedule for the first few versions that use this release model:
 * **Blue** represents the Stabilization phase
 * **Green** represents the Maintenance period
 
-This results in very predictable dates and maintenance periods:
+.. tip::
+
+    If you want to learn more about the timeline of any given Symfony version,
+    use the online `timeline calculator`_. You can also get all data as a JSON
+    string via a URL like `https://symfony.com/roadmap.json?version=2.x`.
```

Comment:
I agree with Wouter. A few days ago we changed the http://symfony.com/roadmap page to add a mention to this TIP:

![symfony_roadmap](https://cloud.githubusercontent.com/assets/73419/13373340/f4b58e42-dd65-11e5-897a-e07478f9c812.png)

Would you agree if I remove this sentence from the `releases.rst` page?